src/lib/wasm/WarpStore.ts

```diff
@@ -55,14 +59,16 @@ class Store {
      * @returns {Promise<wasm.WarpInstance>} A promise that resolves to a WarpInstance.
      * @private
      */
-    private async createIpfs(addresses?: string[]): Promise<wasm.WarpInstance> {
+    async createIpfs(addresses?: string[]): Promise<wasm.WarpInstance> {
         let tesseract: wasm.Tesseract = await TesseractStoreInstance.getTesseract()
         let config: wasm.Config
         if (addresses && addresses.length > 0) {
             config = wasm.Config.minimal_with_relay(addresses)
         } else {
             config = wasm.Config.minimal_basic()
         }
+        // Shuttle address is temporary! For testing!
+        config.enable_shuttle_discovery(["/ip4/138.197.62.183/tcp/7008/ws/p2p/12D3KooW9tAgS6kZPmEtfRiBHsYC46dN2hv6hT6gQ8bqxzWw7y5X"])
```

Comment:
after testing is done i would advise putting this behind some type of dev flag or commenting it out prior to merging